### ISSUE TITLE
false NOTUSED status with EJS

### ISSUE THREAD

**Issue by vctfernandes** [dylang/npm-check#210]
Hey guys,

npm-check is giving me a false alert that EJS is not being used in my project because I never run require('ejs'). Instead, I simply run app.set('view engine', 'ejs').

What do you guys think about a fix?

**Comment by LinusU** [comment]
Yes, I have this problem also, I would love to get some ideas on how we could solve this :)

**Comment by omundy** [comment]
Still a problem in 2022.
![image](https://user-images.githubusercontent.com/279945/148829224-a226ec96-1e84-4f1a-aa10-f44ee03582b4.png)
